### ISSUE TITLE
Readme Correction

### ISSUE THREAD

**Issue by colinluthier** [Digitelektro/MeteorDemod#8]
Not a big deal but on a fresh installation it needs to be 

```git clone https://github.com/Digitelektro/MeteorDemod.git```

instead of:

```
git pull https://github.com/Digitelektro/MeteorDemod.git
cd MeteorDemod
git submodule update --init --recursive
mkdir build && cd build
cmake ../
make
sudo make install
```

**Comment by Digitelektro** [comment]
Thank you, will update and close this issue!

**Comment by Digitelektro** [comment]
Corrected in commit: 3e1d0717b37cdd5dfa20ed7ad4e49a2e293cd841
